### ISSUE TITLE
Support for high quality audio from Go+?

### ISSUE THREAD

**Issue by ithinkandicode** [Tenpi/soundcloud-download#13]
Hello, thank you for making this wonderful extension :)

Does it support the high quality audio that Go+ offers?

I just got a subscription to try it out, and set my [streaming option](https://soundcloud.com/settings/streaming) to "high quality audio", but when I click the download button it downloads an .mp3 file at 128kbps.

So I'm wondering if the extension supports higher quality audio?

Or maybe the original artist only uploaded at 128kbps? I read that SoundCloud only serves high quality files if the artist uploaded a WAV or similar ([source](https://help.soundcloud.com/hc/en-us/articles/360051838074)).

Thanks for your time

**Comment by Tenpi** [comment]
No, it isn't supported. The extension only downloads 128kbps MP3. However I may support the higher quality Go+ audio in the future.
